4.analyze_data/notebooks/UMAP/nbconverted/0.UMAP.py

```diff
@@ -51,12 +62,23 @@
 [cp_dfs[x].shape for x in cp_dfs]
 
 
+# ### Fit UMAP for whole plates
+
 # In[5]:
 
 
 # Fit UMAP features per dataset and save
 for plate in cp_dfs:
+    # Set plate name
     plate_name = pathlib.Path(plate).stem
+    # Set output file for the UMAP
+    output_umap_file = pathlib.Path(output_dir, f"UMAP_{plate_name}.tsv.gz")
```

Comment:
consider parquet for consistancy

Reply:
I would prefer to be consistent, but since this code was developed before our parquet standard, this would take some refactoring. For now, I will plan to keep this as compressed TSV until a full refactor can be done. Thanks for noticing this!